### DISCUSSION TITLE
Make a fork of a private repo public?

### DISCUSSION THREAD

**Discussion by ianisms** [community/community#22624] [New to GitHub]
<p>Looking to see if it would be possible to make a fork of a private repo public.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/ianisms">@ianisms</a>,</p>
<p>Thank you for being here, an organization owner must allow forks of private repositories on the organization level before you can allow or disallow forks for a specific repository. For more information, see “<a href="https://help.github.com/articles/allowing-people-to-fork-private-repositories-in-your-organization" rel="nofollow noopener">Allowing people to fork private repositories in your organization</a>.”</p>
<p>I hope this helps!</p>

**Comment by ianisms** [comment]
<p>Thank you for your reply.  Not excatly what I am looking for.  What I’m looking to discover is specifcally if you can make a fork of a private repo public.  In other words, is it possible to keep the base repo private but allow a fork to go public?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hmmm I understand, private forks can’t be made public. You could try to <a href="https://help.github.com/articles/duplicating-a-repository" rel="nofollow noopener">duplicate the repository</a>.</p>

**Reply by pasanUdr** [reply]
Thank you for the guidance
